scripts/ci/check-regression-promotions.py

```
@@ -167,36 +167,48 @@ def validate(
+        scheduled_selector = scheduled_match.group(1).strip()
+        if scheduled_selector.lower() == "all" or scheduled_selector == "*":
+            scheduled = set(selected)
+        else:
+            scheduled = {
+                case.strip()
+                for case in scheduled_selector.split(",")
+                if case.strip()
+            }
+
+    for case in sorted(scheduled - selected):
+        errors.append(f"scheduled case is not in the harvested inventory: {case}")
```

Ruff (0.16.0)

[warning] Use `list.extend` to create a transformed list

(PERF401)